pkg/controllers/securitypolicy_controller.go

```diff
@@ -17,37 +21,106 @@ import (
 )
 
 var (
-	log = logf.Log.WithName("controller").WithName("securitypolicy")
+	log           = logf.Log.WithName("controller").WithName("securitypolicy")
+	GCInterval    = 60 * time.Second
+	finalizerName = "securitypolicy.nsx.vmware.com/finalizer"
```

Comment:
For `GCInterval` and `finalizerName`, if they won't dynamically change, I prefer to put them in `types.go` as constants.

Reply:
done